tophat/sanity-runner
# feat: catch error and have it printed correctly

Catches lambda invoke error and formats it to printed correctly by our logging functions.

I didnt want to rewrite our entire logging system so i just added a formatFailedResponseResult function that formats the caught invoke error so it is logged/printed correctly. 

For easy debugging i made a container `tophat/sanity-runner:catch-client-error` which is this PR + reduced timeout on invoke to quicker debug. I made a thm-sanity that uses this container to test it catches and prints correctly

quick QA: 
- pull https://github.com/tophatmonocle/thm-sanity/pull/329/files
- `make docker-run-test test=healthcheck`
- should see 1 passing test + 1 failing cause of timeout

slow QA:
- pull this PR
- `cd client`
- `make package`
- update thm-sanity to use this binary 
- run any test
